go/engine/pgp_pull.go

```diff
@@ -207,16 +193,47 @@ func (e *PGPPullEngine) runLoggedIn(m libkb.MetaContext) error {
 		return err
 	}
 
+	pkLookup := make(map[keybase1.UID][]string)
+
+	err = m.G().GetFullSelfer().WithSelf(m.Ctx(), func(user *libkb.User) error {
```

Comment:
Because the UPAK data from list following doesn't include it, need to do a FullSelfer load to get the relevant data now.